src/app/components/Article.jsx

```diff
@@ -1,17 +1,44 @@
-import React, { Fragment } from 'react';
+import React, { Fragment, Component } from 'react';
 import Helmet from 'react-helmet';
+import 'isomorphic-fetch';
 
-const Article = () => {
-  const headline = 'Article Headline';
+class Article extends Component {
+  state = {
+    headline: 'Article Headline',
+  };
 
-  return (
-    <Fragment>
-      <Helmet htmlAttributes={{ lang: 'en-GB' }}>
-        <title>{headline}</title>
-      </Helmet>
-      <h1>{headline}</h1>
-    </Fragment>
-  );
-};
+  static async getInitialProps(context) {
```

Comment:
You can use the destructured syntax here - changing it from `(context)` to `({ req })`. This will make the conditional on line 14-17 much simpler - just an `if(req)`.

Reply:
Huh, I was destructuring and then removed it for some reason. Thanks!